packages/cinema/README.md

```diff
@@ -0,0 +1,51 @@
+# Cinema Package
+
+![](https://media.discordapp.net/attachments/587078058130014240/601166722061434937/cinema.png)
```

Comment:
I don't think we should use external ressources for images, we should store them ourselves.
Also there is no alt, even if it's only in markdown etc, we should still follow the best practices, and set the alt attributes the thing between `[]`.